packages/amplication-data-service-generator/src/tests/__snapshots__/create-data-service.spec.ts.snap

```diff
@@ -3456,7 +3505,7 @@ class Customer {
     each: true,
   })
   @IsOptional()
-  @Field(() => [EnumCustomerFavoriteColors], {
+  @Field(() => EnumCustomerFavoriteColors, {
```

Comment:
This should not change
See the comment in packages/amplication-data-service-generator/src/server/resource/dto/create-field-class-property.ts

Reply:
I think I fixed it. I hope it didn't break something else. Every time I change a condition, it breaks another part.
If nobody wants to refactor this file, I would love to :)